src/utils/dates.test.ts

```diff
@@ -0,0 +1,9 @@
+import { getParsedDate } from './dates';
+
+describe(`getParsedDate`, () => {
+    describe(`calling getParsedDate() with string representation of the date and time`, () => {
```

Comment:
First describe is okey, second I would go with "when called with string represention of the date and time". That way all of this you read: getParsedDate when called with string representation of the date and time should return parsed date in format "dd MMM yyyy". Try to also check other tests for that -> if you can easily read them that way.